### PR TITLE
Create demonstrations.rst

### PR DESCRIPTION
**Pull request by eacharles**
## Change Description

- [ ] My PR includes a link to the issue that I am addressing

## Code Quality
- [ ] I have read the Contribution Guide
- [ ] My code follows the code style of this project
- [ ] My code builds (or compiles) cleanly without any errors or warnings
- [ ] My code contains relevant comments and necessary documentation

### Bug Fix Checklist
- [ ] My fix includes a new test that breaks as a result of the bug (if possible)
- [ ] My change includes a breaking change
  - [ ] My change includes backwards compatibility and deprecation warnings (if possible)

### New Feature Checklist
- [ ] I have added or updated the docstrings associated with my feature using the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html)
- [ ] I have updated the tutorial to highlight my new feature (if appropriate)
- [ ] I have added unit/End-to-End (E2E) test cases to cover my new feature
- [ ] My change includes a breaking change
  - [ ] My change includes backwards compatibility and deprecation warnings (if possible)

### Documentation Change Checklist
- [ ] Any updated docstrings use the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html)

### Build/CI Change Checklist
- [ ] If required or optional dependencies have changed (including version numbers), I have updated the README to reflect this
- [ ] If this is a new CI setup, I have added the associated badge to the README

### Other Change Checklist
- [ ] Any new or updated docstrings use the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html).
- [ ] I have updated the tutorial to highlight my new feature (if appropriate)
- [ ] I have added unit/End-to-End (E2E) test cases to cover any changes
- [ ] My change includes a breaking change
  - [ ] My change includes backwards compatibility and deprecation warnings (if possible)

### CODE REVIEW

File: docs/source/demonstrations.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
+*****************
+Example Notebooks
+*****************
+
+RAIL comes with several notebooks that demonstrate how to use it to analyze data in a number of different ways.
+
+Here we describe the various notebooks and suggest and other in which you might study them.
```
Comment (delucchi-cmu) [inline docs/source/demonstrations.rst]:
```suggestion
Here we describe the various notebooks and suggest other ways in which you might study the data.
```

File: docs/source/demonstrations.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+================================
+
+We recommend starting with the `Goldenspike <https://rail-hub.readthedocs.io/projects/rail-notebooks/en/latest/goldenspike_notebook.html>`_ notebook, 
+which demonstrates a relatively simple end-to-end analysis.  This analysis starts off by making a model that can be used to generate synthetic 
+catalogs of photometric data.  It then uses that model to create sets of synthetic data to train and test per-object redshift estimators, i.e., 
+estimators that compute p(z).  From there it trains and tests a few estimators using some common algorithms.   It then evaluates the 
+performance of those estimators.   Finally, it shows a few methods that converts p(z) for a set of objects to an ensemble distribution n(z).
+
+The `estimation <https://rail-hub.readthedocs.io/projects/rail-notebooks/en/latest/rendered/estimation_examples/RAIL_estimation_demo.html>`_ notebook 
+focus more on the estimation parts of the analysis, and demonstrates a few additional estimation algorithms.
```
Comment (delucchi-cmu) [inline docs/source/demonstrations.rst]:
```suggestion
focuses more on the estimation parts of the analysis, and demonstrates a few additional estimation algorithms.
```

File: docs/source/demonstrations.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+The `evaluation <https://rail-hub.readthedocs.io/projects/rail-notebooks/en/latest/rendered/evaluation_examples/demo.html>`_ of the estimator performance is described in more depth in its own notebook.
+
+Finally, we have collected demonstrates of `useful utilites <https://rail-hub.readthedocs.io/projects/rail-notebooks/en/latest/rendered/core_examples/Useful_Utilities.html>`_ to explore which packages and algorithms are available in the current RAIL installation.
```
Comment (delucchi-cmu) [inline docs/source/demonstrations.rst]:
```suggestion
Finally, we have collected demonstrations of `useful utilites <https://rail-hub.readthedocs.io/projects/rail-notebooks/en/latest/rendered/core_examples/Useful_Utilities.html>`_ to explore which packages and algorithms are available in the current RAIL installation.
```

File: docs/source/demonstrations.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+
+
+Deeper dives into synthetic data creation
+=========================================
+
+The notebooks in  `this <https://rail-hub.readthedocs.io/projects/rail-notebooks/en/latest/creation_notebooks.html>`_ directory demonstate how
```
Comment (delucchi-cmu) [inline docs/source/demonstrations.rst]:
```suggestion
The notebooks in  `this <https://rail-hub.readthedocs.io/projects/rail-notebooks/en/latest/creation_notebooks.html>`_ directory demonstrate how 
```

File: docs/source/demonstrations.rst [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+Finally, we have collected demonstrates of `useful utilites <https://rail-hub.readthedocs.io/projects/rail-notebooks/en/latest/rendered/core_examples/Useful_Utilities.html>`_ to explore which packages and algorithms are available in the current RAIL installation.
+
+
+
+Deeper dives into synthetic data creation
+=========================================
+
+The notebooks in  `this <https://rail-hub.readthedocs.io/projects/rail-notebooks/en/latest/creation_notebooks.html>`_ directory demonstate how 
+how to generate synthetic photometric data, and also how "degrade" the synthetic data by applying various effects to the data.
```
Comment (delucchi-cmu) [inline docs/source/demonstrations.rst]:
```suggestion
how to generate synthetic photometric data, and also how to "degrade" the synthetic data by applying various effects to the data.
```

File: docs/source/demonstrations.rst [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+Finally, `this notebook <https://rail-hub.readthedocs.io/projects/rail-notebooks/en/latest/rendered/estimation_examples/test_sampled_summarizers.html>`_ demonstrates converting collections of per-object p(z) estimates to ensemble n(z) estimates.
+
+
+
+Deeper explanations of rail concepts
+====================================
+
+The two notebooks demonstrate how to `convert a notebook into a ceci analysis pipeline 
+<https://rail-hub.readthedocs.io/projects/rail-notebooks/en/latest/rendered/core_examples/Build_Save_Load_Run_Pipeline.html>`_ and how
```
Comment (delucchi-cmu) [inline docs/source/demonstrations.rst]:
```suggestion
<https://rail-hub.readthedocs.io/projects/rail-notebooks/en/latest/rendered/core_examples/Build_Save_Load_Run_Pipeline.html>`_ and how to
```

**Review by delucchi-cmu — COMMENTED** [review]
My only comments are grammar / spelling nits. These explanations are great!

**Review by olivialynn — APPROVED** [review]
This looks great, thanks for writing this up!

**Review by sschmidt23 — APPROVED** [review]
Looks great, thanks Eric

**Pull request merged**